app.py

```
@@ -4673,6 +4669,15 @@
     return "Done"
 
 
+
+
+@app.route('/health')
+def health_check():
+    try:
+        db.session.execute('SELECT 1')
+        return {'status': 'healthy', 'database': 'connected'}, 200
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)